lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -70,7 +70,7 @@
                     );
                     this.discussionView = new TeamDiscussionView({
                         el: this.$('.discussion-module'),
-                        readOnly: !isMember
+                        readOnly: (!isMember && !isAdminOrStaff)
```

Comment:
@Ali-D-Akbar is there a reason why admin/staff wasn't included here initially? Can we double-check this in documentation!?

Reply:
I just checked documentation is more updated than the code. This `readOnly: !isMember` was written 4 years ago and the documentation is almost 3 years old. Probably left this functionality out back then because it wasn't documented.

So I think there's more authenticity in the docs.

Reply:
"Posts made by discussion community TAs are marked with a Community TA label in the discussion navigation pane, and those made by discussion moderators or discussion admins are marked with a Staff label."

This thing is actually implemented as it is. There was only this an issue with the button that it doesn't show up for admin/staff not being part of a team.